examples/deployer/package.json

```diff
@@ -3,8 +3,9 @@
   "version": "0.1.3",
   "private": true,
   "scripts": {
-    "start": "ts-node src/index.ts",
-    "compile": "ts-node src/compile.ts",
+    "start": "tsx src/index.ts",
+    "compile": "tsx src/compile.ts",
+    "clean": "rimraf ./dist ./.cache",
```

Comment:
Nit: can we use `rm -rf` directly here? Why do we need `rimraf` as a dependency?